tensorflow/compiler/xla/literal_util.cc

```diff
@@ -917,6 +933,16 @@ LiteralUtil::GetMutableArraySlice(Literal* literal) {
 }
 
 template <>
+/* static */ tensorflow::gtl::MutableArraySlice<half>
```

Comment:
or better still, add a 16 bit uint support in protobuf (or even a 16 bit float support in protobuf).  I decided to open an Issue with Protobuf:
https://github.com/google/protobuf/issues/3091